core/src/main/scala/org/apache/spark/ui/JettyUtils.scala

```diff
@@ -209,12 +213,12 @@ private[spark] object JettyUtils extends Logging {
 
       override def filterServerResponseHeader(
           clientRequest: HttpServletRequest,
-          serverResponse: Response,
+          serverResponse: org.eclipse.jetty.client.Response,
```

Comment:
Please use `import` statement. If there is a conflict, you can use Scala's import renaming feature like `JMap` usage.